ReadMe.md

```diff
@@ -164,7 +164,16 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Salma Shaik
 * Prachi Mane
 
-## FAQs
+## General FAQs
```

Comment:
Missing Section heading.

Reply:
Sir I didn't get that. How to do that.

Reply:
I'll fix it...

Reply:
Okay Sir. Sir now can you merge my Pull rerquest. If it seems okay to you.